supplementary_style_guide/style_guidelines/statements-about-the-future.adoc

```diff
@@ -0,0 +1,25 @@
+[[statements-about-the-future]]
+= Statements about the future
+
+[[making-statements-about-future-releases-or-plans]]
+== Making statements about future releases or plans
+
+Regarding future product releases, the _IBM Style Guide_ states, "Avoid claims that are related to the content or timing of a future product or release."  
+
+However, under some limited circumstances (such as _Release Notes_), circumstances will dictate that you refer to a future release, plan, or event. 
+In these situations, follow these guidelines:
+
+- When discussing future plans, use words such as "anticipate," "expect" or "plan."
+- Do not make any promises that a fix for a known issue will ever be included or that a specific feature will be added in an upcoming release.
+- Never refer to a specific future release. Do not refer to a release number or a specific release timeline. 
```

Comment:
Instead of "Never", perhaps simply "Do not" here and in the below bullet?

Reply:
done